### PR TITLE
Update CSS Styling

### PR DESCRIPTION
**Pull request by death7654**
-Make Dark Mode Darker
-Add A Placeholder To The Search Bar In `Supported Devices`
-Match Search Bar Styling to Nav Search Bar in `Supported Devices`
-Make Danger Tip Have Better Contrast
-Make Nav Items Larger for better Accessibility

### CODE REVIEW

**Review by ninelore — CHANGES_REQUESTED** [review]
Just like the other PR: Blocking for Lighthouse checks

**Review by ninelore — CHANGES_REQUESTED** [review]
See above

File: supported-devices/template.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ If your device isn't listed, it doesn't have support.
 :::
 
 ::: tip
-If you're on a smaller screen, scroll sideways to see whole table.
+If you are on a smaller screen, scroll sideways to see whole table.
 :::
 
 <AddScript script-url="../../supported-devices.js"/>
-<p>Search: <input type="text" class="deviceSearch"></p>
+<p><input type="text" class="deviceSearch" placeholder="Enter Your Boardname"></p>
```
Comment (MrChromebox) [inline supported-devices/template.md:43]: this implies that the search function only searches for board names, but that's not the case

Comment (death7654) [reply supported-devices/template.md:43]: The search input in `docs/firmware/supported-devices.html` searches for boardnames only. This is not the search input that is found in the nav
image
image

**Comment by death7654** [comment]
Is it possible to bypass Lint checks for this PR as, it is only a CSS update, and not added content

**Comment by ninelore** [comment]
Checks dont block merge

File: supported-devices/template.md (hunk 1 above)
Comment (ethanaobrien) [reply supported-devices/template.md:43]: No, it checks for boardname and device name

https://github.com/chrultrabook/docs/blob/020da2b85a571336c2473dd2fa2c2d35f4a175e0/supported-devices/search.js#L24

Comment (death7654) [reply supported-devices/template.md:43]: I see, I will update the PR with the changes

Comment (death7654) [reply supported-devices/template.md]: Added in latest commit

File: supported-devices/template.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ If you're on a smaller screen, scroll sideways to see whole table.
 :::
 
 <AddScript script-url="../../supported-devices.js"/>
-<p>Search: <input type="text" class="deviceSearch"></p>
+<p><input type="text" class="deviceSearch" placeholder="Enter Your Boardname Or Device Name"></p>
```
Comment (MrChromebox) [inline supported-devices/template.md:43]: removing the 'Search' prefix, coupled with the css color changes, makes it almost impossible to tell that there is a search bar there now:

image

the old version was much better:
image

Comment (death7654) [reply supported-devices/template.md:43]: I can change the bg and make it pop more

**Comment by MrChromebox** [comment]
the CSS changes here overall reduce the contrast between elements in dark mode, making things harder to see. I can't say I'm a fan at all.

File: supported-devices/template.md (hunk 2 above)
Comment (MrChromebox) [reply supported-devices/template.md:43]: it's one example of a larger problem IMO. I don't know what issue this PR is trying to solve

**Comment by death7654** [comment]
Is this a reference to the search bar or the website as a whole?

File: src/.vuepress/styles/index.scss [hunk 3]
```diff
@@ -18,46 +18,50 @@
...
+  --c-text-accent: #67D6ED; //accent in nav
+  --c-text: #e2e2e2; //text for everything including nav
+  --c-tip-bg: rgb(48, 213, 111, 0.1);
+  --border: #25262C;
+  --c-danger-bg: #51010985;
 }
-.sidebar::-webkit-scrollbar {
-  width: 0%;
-  height: 0%;
+
```
Comment (ninelore) [inline src/.vuepress/styles/index.scss:48]: Why no scrollbar (in the first place)? This is a accessibility issue...

**Comment by MrChromebox** [comment]
> Is this a reference to the search bar or the website as a whole?

both. What is the goal of the CSS changes in this PR?

File: supported-devices/template.md (hunk 2 above)
Comment (ninelore) [reply supported-devices/template.md:43]: The field and text are a accessibility issue aswell due to missing contrast

**Review by ninelore — CHANGES_REQUESTED** [review]

**Comment by ninelore** [comment]
Closing because author declared absence in Discord

**Pull request closed without merging**